mflowgen/tile_array/construct.py

```diff
@@ -85,21 +86,24 @@ def construct():
   route        = Step( 'cadence-innovus-route',         default=True )
   postroute    = Step( 'cadence-innovus-postroute',     default=True )
   signoff      = Step( 'cadence-innovus-signoff',       default=True )
-  pt_signoff   = Step( 'synopsys-pt-timing-signoff',    default=True )
-  genlibdb     = Step( 'synopsys-ptpx-genlibdb',        default=True )
+  #pt_signoff   = Step( 'synopsys-pt-timing-signoff',    default=True )
+  #genlibdb     = Step( 'synopsys-ptpx-genlibdb',        default=True )
+  genlib     = Step( 'cadence-genus-genlib',        default=True )
   drc          = Step( 'mentor-calibre-drc',            default=True )
   lvs          = Step( 'mentor-calibre-lvs',            default=True )
   debugcalibre = Step( 'cadence-innovus-debug-calibre', default=True )
   vcs_sim      = Step( 'synopsys-vcs-sim',              default=True )
 
   # Add cgra tile macro inputs to downstream nodes
 
-  dc.extend_inputs( ['Tile_PE.db'] )
-  dc.extend_inputs( ['Tile_MemCore.db'] )
-  pt_signoff.extend_inputs( ['Tile_PE.db'] )
-  pt_signoff.extend_inputs( ['Tile_MemCore.db'] )
-  genlibdb.extend_inputs( ['Tile_PE.db'] )
-  genlibdb.extend_inputs( ['Tile_MemCore.db'] )
+  #dc.extend_inputs( ['Tile_PE.db'] )
+  synth.extend_inputs( ['Tile_PE_tt.lib'] )
+  #dc.extend_inputs( ['Tile_MemCore.db'] )
+  synth.extend_inputs( ['Tile_MemCore_tt.lib'] )
+  #pt_signoff.extend_inputs( ['Tile_PE.db'] )
+  #pt_signoff.extend_inputs( ['Tile_MemCore.db'] )
+  #genlibdb.extend_inputs( ['Tile_PE.db'] )
+  #genlibdb.extend_inputs( ['Tile_MemCore.db'] )
```

Comment:
Seems like you're missing this here:
```
genlib.extend_inputs( ['Tile_PE_tt.lib'] )
genlib.extend_inputs( ['Tile_MemCore_tt.lib'] )
```

Reply:
My understanding is that this node creates the lib files, how are they inputs?

Reply:
This node creates a .lib file for the tile array. We've already created .lib files for each of the tile types, and this node needs them to create the .lib for the full array.